docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+services:
+  mysql:
+    image: jbergstroem/mariadb-alpine
+    container_name: mysql
+    ports:
+      - "3306:3306/tcp"
+    volumes:
+      - ${DATA_ROOT:-.}/db/data:/var/lib/mysql # where database and logs preserve
+      - ${DATA_ROOT:-.}/db/init:/docker-entrypoint-initdb.d # All xxx.sql, xxx.sql.gz will load into MY_DB. All `xxx.sh` will be executed
+    environment:
+      - MYSQL_DATABASE=${MY_DB:-mtest}
+      - MYSQL_CHARSET=${MY_DB_CHARSET:-utf8mb4}
+      - MYSQL_COLLATION=${MY_DB_COLLATION:-utf8mb4_general_ci}
+      - MYSQL_USER=${MY_DB_USER:-mtest}
+      - MYSQL_PASSWORD=${MY_DB_PASSWORD:-mtest} #
+      - MYSQL_ROOT_PASSWORD=${MY_ROOT_PASSWORD:-root} # set the root password
```

Comment:
This would change the default behavior of how the container works today; I think we should assume as few defaults as possible.

Reply:
Just port the environment-variable setting from your Dockerfile, and leave default values.